pkg/controllers/dynakube/monitoredentities/reconciler.go

```diff
@@ -18,38 +20,48 @@ func NewReconciler( //nolint
 	dk *dynakube.DynaKube,
 ) controllers.Reconciler {
 	return &Reconciler{
-		dk:       dk,
-		dtClient: dtClient,
+		dk:           dk,
+		dtClient:     dtClient,
+		timeProvider: timeprovider.New(),
 	}
 }
 
 type Reconciler struct {
-	dk       *dynakube.DynaKube
-	dtClient dynatrace.Client
+	dk           *dynakube.DynaKube
+	dtClient     dynatrace.Client
+	timeProvider *timeprovider.Provider
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context) error {
-	if r.dk.Status.KubernetesClusterMEID == "" {
-		log.Info("start reconciling monitored entities")
+	log.Info("start reconciling monitored entities")
 
-		monitoredEntities, err := r.dtClient.GetMonitoredEntitiesForKubeSystemUUID(ctx, r.dk.Status.KubeSystemUUID)
-		if err != nil {
-			log.Info("failed to retrieve MEs")
+	if !conditions.IsOutdated(r.timeProvider, r.dk, MEIDConditionType) {
```

Comment:
You either have to let a nil meid pass or just not store an empty meid condition. Otherwise your apimonitoring rereconciler implementation won't work, as in the second call the status would still not be set

Reply:
If I understood you correctly, removing the empty me condition will fix this?

Reply:
possibly (?) done in https://github.com/Dynatrace/dynatrace-operator/pull/3541/commits/f02da5dac342c1f5adb3818ea08fc186560da86e

Reply:
Turns out you set the condition status to `false` so would have worked as expected, so my bad 😅 
But still like it better as it is now, otherwise the status condition would have never become true, if there is no monitored entity